spec/runtime/node_spec.cc

```diff
@@ -43,6 +43,12 @@ describe("Node", []() {
       AssertThat(ts_node_pos(array_node).bytes, Equals<size_t>(2));
       AssertThat(ts_node_size(array_node).bytes, Equals<size_t>(25));
 
+      AssertThat(ts_node_start_point(array_node).row, Equals<size_t>(0));
+      AssertThat(ts_node_start_point(array_node).column, Equals<size_t>(2));
+
+      AssertThat(ts_node_end_point(array_node).row, Equals<size_t>(0));
+      AssertThat(ts_node_end_point(array_node).column, Equals<size_t>(27));
+
```

Comment:
I think that since each `ts_node` function does a little bit of work to keep track of nodes' rows and columns, we need to add assertions about `ts_node_start_point` and `ts_node_end_point` to all of these tests.

Could you change the input string on line 15 to contain some line breaks, so that there'll be some non-trivial row/column computations? Maybe turn it into a pretty-formatted multi-line JSON array, like this:

```c
ts_document_set_input_string(
  document,
  "\n"
  "\n"
  "[\n"
  "  123,\n"
  "  false,\n"
  "  {\n"
  "    \"x\": null\n"
  "  }\n"
  "]"
);
```

This will require changing the `ts_node_pos` assertions.